### PR TITLE
🚸(project) add issue and pull request templates

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose
When new developers join the project, they must respect the good practices put in place for issue and pull request creations. The project is now in an advanced state of development so that issues and PRs are frequent.   

## Proposal

To make it easier for new developers to get started with the project, I've added issue and PRs templates already implemented in other FUN projects. This also aims to be content-oriented when creating issues and PRs and to save time on a daily basis. (Templates are the ones used in `richie` and `marsha` projects).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Actual Behavior**
+A clear and concise description of the behavior.
+
+**Steps to Reproduce**
+1. Do this...
+2. Then this...
+3. And then the bug happens!
+
+**Environment**
+- Marsha version:
```
Comment (jmaupetit) [inline .github/ISSUE_TEMPLATE/Bug_report.md:21]:
```suggestion
- Ralph version:
```

File: .github/ISSUE_TEMPLATE/Support_question.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+about: If you have a question 💬, or something was not clear from the docs!
+
+---
+
+<!-- ^ Click "Preview" for a nicer view! ^
+We primarily use GitHub as an issue tracker. If however you're encountering an issue not covered in the docs, we may be able to help! -->
+
+---
+
+Please make sure you have read our [main Readme](https://github.com/openfun/marsha) as well as our [documentation folder](https://github.com/openfun/marsha/tree/master/docs).
```
Comment (jmaupetit) [inline .github/ISSUE_TEMPLATE/Support_question.md:12]:
```suggestion
Please make sure you have read our [main Readme](https://github.com/openfun/ralph) as well as our [documentation folder](https://github.com/openfun/ralph/tree/master/docs).
```

File: .github/ISSUE_TEMPLATE/Support_question.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Also make sure it was not already answered in [an open or close issue](https://github.com/openfun/marsha/issues).
```
Comment (jmaupetit) [inline .github/ISSUE_TEMPLATE/Support_question.md:14]:
```suggestion
Also make sure it was not already answered in [an open or close issue](https://github.com/openfun/ralph/issues).
```

**Comment by quitterie-lcs** [comment]
@jmaupetit Correction made with `ralph` designation

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**